01-DataJoint Basics - Interactive.ipynb

```diff
@@ -1100,7 +1100,7 @@
    "source": [
     "Behavior of join:\n",
     "\n",
-    "1. match the common field(s) of the primary keys in the two tables\n",
+    "1. match the primary keys of common field(s)in the two tables\n",
```

Comment:
The original one conveys the correct message, albeit a bit too confusing. What we meant there is to match the common fields (attributes) between the two primary keys of the two tables:

Reply:
Ah, I understand. I totally misinterpreted it; it makes sense.

Reply:
Thank you.  Fixed. It should be fine now.